cmd/mattermost/commands/webhook.go

```diff
@@ -101,12 +102,17 @@ func listWebhookCmdF(command *cobra.Command, args []string) error {
 		}
 
 		// Fetch all hooks with a very large limit so we get them all.
-		incomingResult := app.Srv.Store.Webhook().GetIncomingByTeam(team.Id, 0, 100000000)
+		incomingResult := make(chan store.StoreResult, 1)
+		go func() {
+			incomingHooks, err := app.Srv.Store.Webhook().GetIncomingByTeam(team.Id, 0, 100000000)
+			incomingResult <- store.StoreResult{Data: incomingHooks, Err: err}
+			close(incomingResult)
+		}()
 		outgoingResult := app.Srv.Store.Webhook().GetOutgoingByTeam(team.Id, 0, 100000000)
 
 		if result := <-incomingResult; result.Err == nil {
-			CommandPrettyPrintln(fmt.Sprintf("Incoming webhooks for %s (%s):", team.DisplayName, team.Name))
 			hooks := result.Data.([]*model.IncomingWebhook)
+			CommandPrettyPrintln(fmt.Sprintf("Incoming webhooks for %s (%s):", team.DisplayName, team.Name))
```

Comment:
nit: Does this line need to be moved? It would make the (long-term) diff simpler, and avoid splicing the `hooks` declaration with its population.

Reply:
definitely no. i just coped from `outgoingHooks` section during my revision. changing it back.